osc-server/src/main/java/org/osc/core/broker/service/validator/BindSecurityGroupRequestValidator.java

```diff
@@ -31,14 +31,15 @@ public static void checkForNullFields(BindSecurityGroupRequest request) throws E
 
         // build a map of (field,value) pairs to be checked for null/empty
         // values
-        Map<String, Object> map = new HashMap<String, Object>();
+        Map<String, Object> map = new HashMap<>();
 
         map.put("Security Group Id", request.getSecurityGroupId());
         List<VirtualSystemPolicyBindingDto> services = request.getServicesToBindTo();
         if (services != null && !services.isEmpty()) {
             for (VirtualSystemPolicyBindingDto service : services) {
-                if (service.getVirtualSystemId() == null || service.getPolicyId() == null
-                        || StringUtils.isBlank(service.getName()) || service.getOrder() == null) {
+				if (service.getVirtualSystemId() == null || service.getPolicyIds() == null
+						|| service.getPolicyIds().isEmpty() || StringUtils.isBlank(service.getName())
```

Comment:
not sure if this condition makes sense here. Policy ids can be empty/null in case manager does not support policy mapping